exonum-java-binding/tutorials/car-registry/README.md

```diff
@@ -0,0 +1,38 @@
+
+# car-registry
```

Comment:
This is a generated readme, probably, shall be replaced with a link to the tutorial, once it is published.